docs/using-tasks-to-prepare-the-environment/session-config.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Session config
+weight: 4
```

Comment:
Here, I didn't know what the weight should be. So, it's just a speculative value.